README.md

```diff
@@ -1 +1,27 @@
-# OOP-School-library
\ No newline at end of file
+# OOP-School-
+
+An imaginary library that helps store books, and give access to a particuar age group and seek permission for student below a particular age.
+
+# Getting started
+
+- ruby install
+- clone repo from [@githubhandle](https://github.com/SirriRyisa)
```

Comment:
- It's better to explicitly add the command to be used for cloning. Makes it easier to start with the project